cmd/internal/cli/delete.go

```diff
@@ -82,17 +95,19 @@ var deleteImageCmd = &cobra.Command{
 			Logger:    (golog.Logger)(sylog.DebugLogger{}),
 		}
 
-		y, err := interactive.AskYNQuestion("n", "Are you sure you want to delete %s arch[%s] [N/y] ", imageRef, deleteImageArch)
-		if err != nil {
-			sylog.Fatalf(err.Error())
-		}
-		if y == "n" {
-			return
+		if !deleteForce {
+			y, err := interactive.AskYNQuestion("n", "Are you sure you want to delete %s arch[%s] [N/y] ", imageRef, deleteImageArch)
```

Comment:
Can we please have a space between `arch` and `[%s]`?

Reply:
How about just `(AMD64)` instead of `arch [amd64]` ?